build/charts/theia/values.yaml

```diff
@@ -144,12 +144,15 @@ grafana:
   # -- The dashboards to be displayed in Grafana UI. The files must be put under
   # provisioning/dashboards.
   dashboards:
+    - homepage.json
     - flow_records_dashboard.json
     - pod_to_pod_dashboard.json
     - pod_to_service_dashboard.json
     - pod_to_external_dashboard.json
     - node_to_node_dashboard.json
     - networkpolicy_dashboard.json
+    # -- Path to the default home dashboard.
+  homeDashboard: /opt/grafana/dashboards/homepage.json
```

Comment:
I'm not sure whether it would be confusing to include "/opt/grafana/dashboards/" here. As this path is hard coded in templates and does not expose to users, maybe it would be better to let the user only provide the default home dashboard name which is `homepage.json` here.

Reply:
make sense to me. changed as suggested, thanks